src/librbd/cache/pwl/AbstractWriteLog.cc

```diff
@@ -826,12 +826,6 @@ void AbstractWriteLog<I>::write(Extents &&image_extents,
 
   ceph_assert(m_initialized);
 
-  /* Split images because PMDK's space management is not perfect, there are
-   * fragment problems. The larger the block size difference of the block,
-   * the easier the fragmentation problem will occur, resulting in the
-   * remaining space can not be allocated in large size. We plan to manage
-   * pmem space and allocation by ourselves in the future.
-   */
```

Comment:
Is this comment really outdated?  IIUC you are still planning to switch from libpmemobj to raw libpmem and therefore manual allocation management?

Reply:
> Is this comment really outdated? IIUC you are still planning to switch from libpmemobj to raw libpmem and therefore manual allocation management (#41432)?

I think it is outdated. I have the plan to manage pmem space by libpmem. But PMDK libpmemobj don't have fragment problems now. It fixed in https://github.com/ceph/ceph/pull/40493 by new method. Therefore, in fact, the split operation is not necessary, but in order to make it easier to allocate space by libpmemobj and reduce the defragmentation operation, this step is still beneficial. If you think it necessary to explain, the comment should be changed to "split blocks to make it easier to allocate space, reduce pmem's defragmentation operations, and plan to use libpmem to manage space by itself in the future"

Reply:
How about
```
  /*
   * Split image extents larger than 1M.  This isn't strictly
   * necessary but makes libpmemobj allocator's job easier and
   * reduces pmemobj_defrag() cost.  We plan to manage pmem space
   * and allocation by ourselves in the future.
   */
```

Reply:
done